pages/docs/specifications/latest.md

```diff
@@ -0,0 +1,2188 @@
+# AsyncAPI Specification
```

Comment:
we should not create `latest.md` but modify the redirect that you removed to always point to the latest spec. As a side effect you do not have a problem with the title of the document in the UI mentioned by @magicmatatjahu

Reply:
I thought you wanted to solve it differently than with github action :) now it should stay

Reply:
I think it better is to do it with redirect than duplicate content